workshops/forms.py

```diff
@@ -88,17 +88,8 @@ class BootstrapHelperFormsetInline(BootstrapHelper):
 bootstrap_helper_inline_formsets = BootstrapHelperFormsetInline()
 
 
-class InstructorsForm(forms.Form):
+class WorkshopStaffForm(forms.Form):
     '''Represent instructor matching form.'''
-
-    latitude = forms.FloatField(label='Latitude',
-                                min_value=-90.0,
-                                max_value=90.0,
-                                required=False)
-    longitude = forms.FloatField(label='Longitude',
-                                 min_value=-180.0,
-                                 max_value=180.0,
-                                 required=False)
```

Comment:
It was suggested by Tracy (and I feel like she's right) that these fields aren't used. I can understand that it's cumbersome to, for example, find a place on Google Maps, get the coordinates and enter them in this form.